src/modules/map/i18n/index.js

```diff
@@ -1,5 +1,7 @@
 
 import { provide } from './map.provider';
+import { layerManagerProvide } from './layerManager.provider';
```

Comment:
The naming here was a starting point from me, but i would prefer a balanced naming for both provide-methods, like  
`mapProvide` and `layerManagerProvide`. 
With the growing of components in the modules with they seperated i18n-providers, there should be a common convention how to deal with it.

Reply:
thx